docs/development/demo.md

```diff
@@ -0,0 +1,40 @@
+# Synapse demo setup
+
+**DO NOT USE THESE DEMO SERVERS IN PRODUCTION**
+
+Requires you to have a [Synapse development environment setup](https://matrix-org.github.io/synapse/develop/development/contributing_guide.html#4-install-the-dependencies).
+
+The demo setup allows running three federation Synapse servers, with server
+names `localhost:8080`, `localhost:8081`, and `localhost:8082`.
+
+You can access them via any Matrix client over HTTP at `localhost:808{1,2,3}` or
```

Comment:
Forgive me for my confusion, but I don't quite understand this part and thought I'd ask! The doc says that the script starts three servers with the names `localhost:8080`, `localhost:8081`, and `localhost:8082`. However, on the next line it says to access them at `localhost:808{1,2,3}`. My first thought upon looking that line was that this was saying to access them at `localhost:8081`, `localhost:8082`, or `localhost:8083` which I don't think is correct? I think beginners might take the `{1,2,3}` literally rather than reading it as a set of variables referring to `localhost:8080`, `localhost:8081`, etc. which is what I am assuming you meant?

Reply:
> However, on the next line it says to access them at `localhost:808{1,2,3}`. My first thought upon looking that line was that this was saying to access them at `localhost:8081`, `localhost:8082`, or `localhost:8083` which I don't think is correct?

That is what this line is trying to say (and is correct). I can expand it out.

Reply:
I updated this (and another instance) in d46c39b5dcbc02844d3980731be62438b8bb57ff.